searchEngine/testing/searchTest.py

```diff
@@ -0,0 +1,69 @@
+import os
+import sys
+import inspect
+
+current_dir = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
+parent_dir = os.path.dirname(current_dir)
+sys.path.insert(0, parent_dir) 
+from SearchEngine import SearchEngine
+from MockArticle import Article
+
+# Assuming articles already come in lemma form, lemma form created manually using cloud natural language
+article_1 = Article(1, 1,"El conductor y ocupante de bicicleta, bicimoto o triciclos poder utilizar de preferencia casco de proteccion para el seguridad.", 
```

Comment:
This example suggests to me that, especially in a language such as Spanish, you will probably need to expand your definition of keywords to include [bigrams and trigrams][]. In this example, `casco de proteccion` seems like an important keyword, but your current approach to the prototype does not represent it.

[bigrams and trigrams]: https://towardsdatascience.com/understanding-word-n-grams-and-n-gram-probability-in-natural-language-processing-9d9eef0fa058

Reply:
This is something I sort of mentioned in our original design doc. I mentioned context analysis, not n grams, but a close enough concept. I'm not sure if it fits the scope of this project especially since there are so many possible improvements the search algorithm needs, but I will add it to possible improvements.

Reply:
Apparently context analysis uses n grams. https://www.lexalytics.com/lexablog/context-analysis-nlp